src/scripts/main.js

```diff
@@ -1,7 +1,21 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const message = document.createElement('div');
+
+  message.classList.add('notification', type);
+
+  message.innerHTML = `<h2 class="title">${title}</h2>
+    <p>${description}</p>`;
+
+  message.style.top = `${posTop}px`;
+  message.style.right = `${posRight}px`;
+
+  document.body.append(message);
+
+  setTimeout(() => {
+    message.style.display = 'none';
```

Comment:
you should remove the element, do not change the display property

Reply:
yes, i had tried it before, but had problem with tests